AttestationPoint.md

```diff
@@ -0,0 +1,108 @@
+# Deterministic Attestation Point Computation
+
+## Introduction
+
+Discreet Log Contracts (DLCs) are oracle contracts enforced using a set of adaptor signature "point-locks."
+This means that in order to spend any execution branch of a DLC, one must reveal the scalar pre-image for a point.
+As is described in this document, the points used for adaptor signing correspond to anticipations of all
+possible oracle attestations such that exactly one adaptor secret is revealed, unlocking exactly one signature,
+making exactly one Contract Execution Transaction (CET) valid for on-chain publication.
```

Comment:
nit: it might be better to link to the glossary than to redefine acronyms every time?